docs/src/bpmn-workflows/receive-tasks/receive-tasks.md

```diff
@@ -5,18 +5,18 @@ Receive tasks are tasks which references a message. They are used to wait until
 ![Receive Tasks](/bpmn-workflows/receive-tasks/receive-tasks.png)
 
 When a receive task is entered then a corresponding message subscription is created. The workflow instance stops at this point and waits until the message is correlated.
- 
-A message can published using one of the Zeebe clients. When the message is correlated, the receive task gets completed and the workflow instance continues. 
+
+A message can published using one of the Zeebe clients. When the message is correlated, the receive task gets completed and the workflow instance continues.
 
 > An alternative to receive tasks are [message intermediate catch events](/bpmn-workflows/message-events/message-events.html) which behaves the same but can be used together with event-based gateways.
 
 ## Messages
 
-A message can be referenced by one or more receive tasks. It **must** define the name of the message (e.g. `Money collected`) and the `correlationKey` variable (e.g. `orderId`).
+A message can be referenced by one or more receive events. It **must** define the name of the message (e.g. `Money collected`) and the `correlationKey` expression (e.g. `= orderId`).
```

Comment:
You've changed the name here to receive events, but the rest of this document refers to them as receive tasks.